### PR TITLE
Workaround lighthouse build

### PR DESCRIPTION
**Pull request by rajsite**
# Pull Request

## 🤨 Rationale

Disables the Card in the Angular and Blazor example apps as it breaks axe-core tooling used by lighthouse, see: https://github.com/ni/nimble/issues/1650

## 👩‍💻 Implementation

Disabled the nimble-card in the Angular and Blazor apps

Re-enabled the accessibility lighthouse error. The thought is that with this change the accessibility score will be reliably calculated and not cause intermittent blocking of merges, only blocks on merges that further regress the score. If it is not the case and lighthouse becomes unreliable calculating the score we can disable it again.

Removed the exisiting comment for an issue to get us back to 0.9 score as there are multiple issues impacting our score now.

## 🧪 Testing

Relying on lighthouse CI execution.

## ✅ Checklist

- [x] I have updated the project documentation to reflect my changes or determined no changes are needed.

### CODE REVIEW

**Comment by rajsite** [comment]
@mollykreis Could you review this PR for re-enabling the lighthouse accessibility check (and if that seems like a good idea)

File: angular-workspace/lighthouserc.js [hunk 1]
```diff
@@ -16,9 +16,7 @@ module.exports = {
         assert: {
             assertions: {
                 'categories:performance': ['warn', { minScore: 0.9 }],
-                // TODO: reset to 0.9 as part of https://github.com/ni/nimble/issues/1090
-                // TODO: reset to error as part of https://github.com/ni/nimble/issues/1650
-                'categories:accessibility': ['warn', { minScore: 0.88 }]
+                'categories:accessibility': ['error', { minScore: 0.8 }]
```
Comment (mollykreis) [inline angular-workspace/lighthouserc.js:19]: > Removed the exisiting comment for an issue to get us back to 0.9 score as there are multiple issues impacting our score now.

Do we want to bring this number back up eventually? If so, any ideas on how we should remember to do that?

**Review by mollykreis — APPROVED** [review]

File: angular-workspace/lighthouserc.js (hunk 1 above)
Comment (rajsite) [reply angular-workspace/lighthouserc.js:19]: I guess that is covered in https://github.com/ni/nimble/issues/280 which states a more ambitious 1.0 goal

**Comment by rajsite** [comment]
Bypassing owners (mostly OOO) and buddied with Molly

**Pull request merged**